From 3c2ae1280db02b56227762b9dff2b088892f413f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dagfinn=20Ilmari=20Manns=C3=A5ker?= <ilmari@ilmari.org>
Date: Thu, 5 Feb 2015 21:37:08 +0000
Subject: Fix setenv() replacement documentation in perlclib

my_putenv doesn't exist, the function is called my_setenv

(cherry picked from commit 7ad03f5069e771fe44e92b694a23f10354c548d1)

Bug-Debian: https://bugs.debian.org/822336
Patch-Name: fixes/5.20.3/docs/setenv.diff
---
 pod/perlclib.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pod/perlclib.pod b/pod/perlclib.pod
index 0cdee2491..2e2078439 100644
--- a/pod/perlclib.pod
+++ b/pod/perlclib.pod
@@ -222,7 +222,7 @@ everywhere by now.
  system(s)                   Don't. Look at pp_system or use my_popen
 
  getenv(s)                   PerlEnv_getenv(s)
- setenv(s, val)              my_putenv(s, val)
+ setenv(s, val)              my_setenv(s, val)
 
 =head2 Miscellaneous functions
 
